.github/workflows/odh-manifests-PR-sync.yml

```diff
@@ -0,0 +1,70 @@
+name: odh-manifests sync
+
+run-name: Sync manifests in odh-manifests
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  send-pull-requests:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout data-science-pipelines-operator repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: main
+          repository: DharmitD/data-science-pipelines-operator
```

Comment:
Repo doesn't seem right

Reply:
Done, updated repo to `opendatahub-io/data-science-pipelines-operator`

Reply:
Did you not push it up?

Reply:
Yes, my bad, I missed out on pushing it up earlier, had it updated to point to odh repos in my local.

But it should be updated now, with this push: https://github.com/opendatahub-io/data-science-pipelines-operator/compare/449a0f281edf2bdeb65f37ad3a95fd3f152a5ea3..ddb44c76e977df0046cbf1a811cf4f3915c0b84d